backend/api/Database/Models/InspectionFindings.cs

```diff
@@ -1,22 +1,42 @@
-﻿using Microsoft.EntityFrameworkCore;
-
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Api.Controllers.Models;
 #pragma warning disable CS8618
 namespace Api.Database.Models
 {
-    [Owned]
     public class InspectionFindings
     {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public string Id { get; set; }
 
         public string RobotName { get; set; }
 
         public string InspectionDate { get; set; }
```

Comment:
Should perhaps be a DateTime object

Reply:
Yes I think it should be. Also highlight the correct parameter because this got me thinking why should I have "ID" as date time object :P . Now the InspectionDate is datetime object. =) .